src/getHumanAge.test.js

```diff
@@ -7,4 +7,37 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return 0, if animalAge less than first', () => {
```

Comment:
The test description mentions 'animalAge less than first', but it should specify that both `catAge` and `dogAge` are less than 15, which is the threshold for the first human year according to the task description.